src/ontology/mondo-edit.obo

```diff
@@ -555140,12 +555159,16 @@ subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_disorder {source="Orphanet:659873"}
 subset: orphanet_rare {source="Orphanet:659873"}
 subset: rare
+synonym: "LEMD2-associated nuclear envelopathy with early progeroid appearance" EXACT [https://orcid.org/0000-0001-9310-0163]
+synonym: "Marbach-Rustad progeroid syndrome" EXACT [https://orcid.org/0000-0001-9310-0163]
+synonym: "Wormian bones-micrognathia-abnormal dentition-progeroid syndrome" EXACT [https://orcid.org/0000-0001-9310-0163]
```

Comment:
Adding the orphanet x-ref for all of these synonyms/ 
Might be added automatically via our synonym sync pipeline, but I want to make sure it is there by the time we release